src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -136,6 +136,4 @@ private ModelAndView createErrorModelAndView (String errorTitle, String errorDet
         return modelAndView;
     }
 
-
-
```

Comment:
Removed excess whitespace for a cleaner look
https://www.baeldung.com/java-clean-code